openwisp_users/apps.py

```diff
@@ -23,16 +23,26 @@ class OpenwispUsersConfig(AppConfig):
     verbose_name = _('Users and Organizations')
 
     def ready(self):
-        self.register_default_menu_items()
+        self.register_menu_group()
         self.set_default_settings()
         self.connect_receivers()
 
-    def register_default_menu_items(self):
-        register_menu_items(
-            [
-                {'model': settings.AUTH_USER_MODEL},
-                {'model': get_model_name('openwisp_users', 'Organization')},
-            ]
+    def register_menu_group(self):
+        register_menu_group(
+            position=4,
+            config={
+                'label': 'User',
+                'model': settings.AUTH_USER_MODEL,
+                'name': 'changelist',
+            },
+        )
+        register_menu_group(
+            position=5,
+            config={
+                'label': 'Organizations',
```

Comment:
flag string as translatable, always do this for strings which are shown in the UI